docs/src/main/sphinx/installation/cli.rst

```diff
@@ -46,17 +46,31 @@ The recommended TLS implementation is to use a globally trusted certificate. In
 this case, no other options are necessary, since the JVM running the CLI
 recognizes these certificates.
 
+Use ``--help`` to see information about specifying the keystore, truststore, and
+other authentication details as required.
+
 Authentication
 --------------
 
-You can override your username with the ``--user`` option. It defaults to your
-operating system username. If your Trino server requires password
-authentication, use the ``--password`` option to have the CLI prompt for a
-password. You can set the ``TRINO_PASSWORD`` environment variable with the
-password value to avoid the prompt.
+The Trino CLI supports many authentication methods detailed in the following
+sections:
 
-Use ``--help`` to see information about specifying the keystore, truststore, and
-other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
+Username and password authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Usermane and password authentication is typically configured in a cluster with
+the ``PASSWORD`` authentication type. Password file authentication is very
```

Comment:
just link to the authentication type doc and get rid of the sentence "Password file ..."